.github/workflows/api-test.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           ./mvnw -B clean install \
           -Dmaven.test.skip \
           -Dmaven.javadoc.skip \
-          -Dmaven.checkstyle.skip \
+          -Dcheckstyle.skip=true \
```

Comment:
Should we skip checkstyle in ```e2e.yaml```?

Reply:
I think we can skip, we need to add a single checkstyle task, this need to wait we solve the all exist code format errors.

Reply:
> I think we can skip, we need to add a single checkstyle task, this need to wait we solve the all exist code format errors.

That would be great.